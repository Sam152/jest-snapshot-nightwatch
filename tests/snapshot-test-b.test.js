const expect = require('expect');
const toMatchSnapshot = require('../index');

expect.extend({toMatchSnapshot});

module.exports = {
  '@unitTest': true,
  'Second test using snapshots': function () {
    expect('foo').toMatchSnapshot();
    expect('bar').toMatchSnapshot();
  },
};
