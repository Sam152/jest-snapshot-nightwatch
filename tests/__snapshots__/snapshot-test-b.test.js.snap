// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Second test using snapshots 1`] = `"foo"`;

exports[`Second test using snapshots 2`] = `"bar"`;

exports[`Second test using snapshots 3`] = `"baz"`;
